### PR TITLE
DL-3886 play 2.7 upgrade

### PR DESCRIPTION
**Pull request by juamalBlackman**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -6,17 +6,16 @@ private object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-26"             % "1.8.0",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-27"    % "2.21.0",
     "uk.gov.hmrc" %% "simple-reactivemongo"          % "7.30.0-play-26",
```
Comment (emmaatkins) [inline project/AppDependencies.scala:10]: This is still play 26?

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -6,17 +6,16 @@ private object AppDependencies {
...
     "uk.gov.hmrc" %% "logback-json-logger"           % "4.8.0",
-    "uk.gov.hmrc" %% "govuk-template"                % "5.55.0-play-26",
-    "uk.gov.hmrc" %% "play-health"                   % "3.15.0-play-26",
-    "uk.gov.hmrc" %% "play-ui"                       % "8.11.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client"           % "9.1.0-play-26",
+    "uk.gov.hmrc" %% "govuk-template"                % "5.55.0-play-27",
+    "uk.gov.hmrc" %% "play-health"                   % "3.15.0-play-27",
+    "uk.gov.hmrc" %% "play-ui"                       % "8.11.0-play-27",
+    "uk.gov.hmrc" %% "http-caching-client"           % "9.1.0-play-27",
     "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.2.0-play-26",
```
Comment (emmaatkins) [inline project/AppDependencies.scala:16]: This one too?

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -6,17 +6,16 @@ private object AppDependencies {
...
-    "uk.gov.hmrc" %% "play-language"                 % "4.3.0-play-26",
-    "uk.gov.hmrc" %% "play-partials"                 % "6.11.0-play-26",
-    "uk.gov.hmrc" %% "domain"                        % "5.9.0-play-26"
+    "uk.gov.hmrc" %% "play-language"                 % "4.3.0-play-27",
+    "uk.gov.hmrc" %% "play-partials"                 % "6.11.0-play-27",
```
Comment (emmaatkins) [inline project/AppDependencies.scala:18]: comma needs to go

Comment (CEPoole) [reply project/AppDependencies.scala:18]: Not actually true in newest Scala/Play but I'd still like it gone too 😬

File: project/AppDependencies.scala (hunk 1 above)
Comment (benryan1790) [reply project/AppDependencies.scala:10]: Also you only need bootstrap-frontend-play-27, bootstrap-play-26 can go

Comment (juamalBlackman) [reply project/AppDependencies.scala]: changes made

**Pull request merged**